src/components/yeti/SubPanelTitle.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div class="columns is-mobile">
     <div class="column">
-      <h2 class="title is-3">
+      <h2 class="title is-3" v-translate>
```

Comment:
This cannot be used because key extraction will fail. That why I introduced the change on https://github.com/c2corg/c2c_ui/commit/b3246ff7249fb4a0b3f277b139d53440d9c1887c

Reply:
Ha, OK, sorry for that. 

The point is there are notices in the console
```

gettext-plugin.js:209 v-translate must contains only text NodeList [div.column]
```

Reply:
You're right. I did another fix for this, which requires an extra `<span>` tag. I also included your date fix in https://github.com/c2corg/c2c_ui/commit/41be420233058181e21bab3b6b349d05b6cfc1be.

So closing this PR, I'll probably do a release today or on thursday.